FrameRate/FrameRate.cpp

```diff
@@ -35,6 +38,8 @@
 
 // Events
 #define EVENT_FPS_UPDATE "onFpsEvent"
+#define EVENT_FRAMERATE_PRECHANGE  "onDisplayFrameRateChanging"
```

Comment:
Pls document these events. You can refer to instructions in documentation section under the main README.